src/SprykerEco/Yves/Easycredit/Handler/EasycreditPaymentHandler.php

```diff
@@ -22,7 +22,7 @@ class EasycreditPaymentHandler implements EasycreditPaymentHandlerInterface
      */
     public function addPaymentToQuote(QuoteTransfer $quoteTransfer): AbstractTransfer
     {
-        $quoteTransfer->getPayment()
+        $quoteTransfer->getPaymentOrFail()
```

Comment:
Please check all cases where you add `get*OrFail()` and update facade method specifications if transfer object is passed through it.